invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/ActionDropdown.js

```diff
@@ -77,15 +77,15 @@ class ActionDropdown extends Component {
             onChange={this.handleOnChange}
             disabled={disabled}
           />
-          {actionSuccess && (
-            <SuccessIcon
-              timeOutDelay={4000}
-              show={actionSuccess}
-              className="ml-10"
-            />
-          )}
-          {error && <ErrorPopup error={error} />}
-        </>
+          <div className="small-reserved-space">
```

Comment:
Not sure about this class name?

Reply:
where is this class defined?

Reply:
Be aware that I removed this class in https://github.com/inveniosoftware/invenio-app-rdm/pull/1538. Can we avoid having this custom class?

Reply:
i was not sure either, in any case i added it so that the dropdown will always keep some room left for the success and error icons. Otherwise it will wrap under it which does not look nice at all. I tried to use a grid layout but it didn't have the wanted behaviour (dropdown does not adhere to the  column width)

Reply:
looked at it with @kpsherva and found that setting a bigger column width achieves similar results without the class :raised_hands: